src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -0,0 +1,403 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, division, absolute_import
+
+#
+# Subscription manager command line utility.
+#
+# Copyright (c) 2010 Red Hat, Inc.
```

Comment:
We should probably update these copyright dates to be current, while we're in here.